src/sentry/seer/anomaly_detection/get_anomaly_data.py

```diff
@@ -153,3 +155,112 @@ def get_anomaly_data_from_seer(
         )
         return None
     return ts
+
+
+def get_anomaly_data_from_seer(
+    sensitivity,
+    seasonality,
+    threshold_type,
+    subscription: QuerySubscription,
+    subscription_update: MetricDetectorUpdate,
+    source_id,
+) -> list[TimeSeriesPoint] | None:
+    snuba_query: SnubaQuery = subscription.snuba_query
+    aggregation_value = subscription_update["values"]["value"]
+
+    extra_data = {
+        "subscription_id": subscription.id,
+        "organization_id": subscription.project.organization.id,
+        "project_id": subscription.project_id,
+        "source_id": source_id,
+    }
+
+    anomaly_detection_config = AnomalyDetectionConfig(
+        time_period=int(snuba_query.time_window / 60),
+        sensitivity=sensitivity,
+        direction=translate_direction(threshold_type),
+        expected_seasonality=seasonality,
+    )
+    context = AlertInSeer(
+        id=source_id,
```

Comment:
```suggestion
        id=source_id.get("data_packet_source_id"),
```

Reply:
Or am I confused 🤔 because `source_id` is a dict
```
source_id = {
    "data_packet_source_id": update.source_id,
    "data_packet_type": data_source.type,
}